### DISCUSSION TITLE
Best way to send mobile phone notification with images?

### DISCUSSION THREAD

**Discussion by ImLovinIt** [blakeblackshear/frigate#2327] [Ask A Question] [answered]
I've been testing Frigate for a while now and love it. My goal now is to set up image notifications while I'm away.

1. Not keen to expose my HA to the internet for remote access. Just not confident with my skills yet.
2. When connected to home wifi, my phone can receive images with no problem. When away, only the text part can be seen. Understand it's the local image issue and only can be accessed locally.
3. Tried Discord notification. It seems only to be capable of sending images with a local path to the discord server channel?
    - config/www/test.jpg ### works for me on discord
    - "http://homeassistant.local:8123/api/frigate/notifications/{{trigger.payload_json['after']['id']}}/thumbnail.jpg" ### works with HA app (notify.notify) but not discord channel
4. Haven't tried Telegram. Please share your experience.

How do you guys receive image notifications on your mobile? Would prefer to see an image preview on my mobile notification.

**Comment by bm55** [comment, marked as answer]
I send the snapshot from frigate to my Telegram

**Reply by Minglarn** [reply]
Telegram or Pushover is recommended 
Tried the HA app but the images vanished into space when open a notification. 

Use Telegram or Pushover as you get a scroll list with all the events available fast and secure. You can even add a link to the recorded video so you can watch the event right away.

**Reply by ImLovinIt** [reply]
Thanks. Telegram works perfectly. My only concern is the bot is visible to the rest of the world? Is there a password protection method? Whoever searches bots in Telegram might be able to add my bot right? Is there a way to make it hidden?

**Reply by ImLovinIt** [reply]
I think I figured it out. When configuring the telegram bot, I forgot I typed in the Chat ID. This should prevent others from receiving my notifications even if they add my bot. The only thing is when passing URL to the bot, "...homeassistant.local:8123..." didn't work on my local network. Had to type in HA's actual local IP address.

After playing around with Discord and Telegram, I decide to go with Telegram for its security and dedicate Discord to gaming purpose only. Plus Telegram has the option on the mobile phone to automatically clear cache after certain days and set cache capped at a certain size.

**Comment by mcdent** [comment]
Pretty sure if you subscribe to the Nabu Casa remote service, then push notifications will work on local wifi or away from local wifi.
